src/Features/Core/SolutionCrawler/WorkCoordinator.HighPriorityProcessor.cs

```diff
@@ -114,7 +114,7 @@ protected override Task WaitAsync(CancellationToken cancellationToken)
                             // see whether we have work item for the document
                             WorkItem workItem;
                             CancellationTokenSource documentCancellation;
-                            Contract.ThrowIfFalse(GetNextWorkItem(out workItem, out documentCancellation));
+                            Contract.ThrowIfFalse(TryGetNextWorkItem(out workItem, out documentCancellation));
```

Comment:
other case, it has input like documentId and projectId, and we might not have workitem for the given input, but this case, it says any workItem, and there must be one. let me re-name method.